mesonbuild/interpreter.py

```diff
@@ -2177,10 +2177,7 @@ def func_subdir(self, node, args, kwargs):
                                    % subdir)
         self.visited_subdirs[subdir] = True
         self.subdir = subdir
-        try:
-            os.makedirs(os.path.join(self.environment.build_dir, subdir))
-        except FileExistsError:
-            pass
+        os.makedirs(os.path.join(self.environment.build_dir, subdir), exist_ok = True)
```

Comment:
According to PEP 8, and in accordance with the rest of the codebase, there is no space between keyword arguments and the value. Please change this to `exist_ok=True`.